src/js/api/config.js

```diff
@@ -84,6 +84,13 @@ const Config = function(options, persisted) {
     config.language = language;
     config.intl = intl;
 
+    // If autoPause is configured with an empty block,
+    // default autoPause.viewability to true.
+    let autoPause = config.autoPause;
+    if (autoPause && !autoPause.hasOwnProperty('viewability')) {
```

Comment:
`'viewability' in autoPause`. See https://github.com/jwplayer/jwplayer/pull/3282#discussion_r257038288